From 6f5265b3f90c711aee64144aab390291685305a9 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Mon, 06 May 2024 12:55:01 +0300
Subject: [PATCH] [FSW-402] fs1: init: fix progress bits sequence

this patch add fix to correct progress bits
sequence, to avoid wastage of one bit in software
status register bits.

Change-Id: I25dce155e0c392638889c0194ec7e2ab62bec82f
---

diff --git a/arch/fs1/arc/progress_bits.h b/arch/fs1/arc/progress_bits.h
index 3efae37..780db47 100644
--- a/arch/fs1/arc/progress_bits.h
+++ b/arch/fs1/arc/progress_bits.h
@@ -73,57 +73,57 @@
 	PROGRESS_BOOT_ROM_TPM = 63,
 	PROGRESS_BOOT_ROM_TPM_INIT_FAIL = 64,
 	PROGRESS_BOOT_ROM_TPM_STARTUP_FAIL = 65,
-	PROGRESS_BOOT_ROM_TPM_PCR_EXTEND_FAIL = 67,
-	PROGRESS_BOOT_ROM_TPM_FINISH = 68,
-	PROGRESS_PPBOOT_START = 69,
-	PROGRESS_PPBOOT_FINISH = 70,
-	PROGRESS_BOOT_ROM_IN_RECOVERY = 71,
-	PROGRESS_BOOT_ROM_IN_ROM_RECOVERY = 72,
-	PROGRESS_BOOT_ROM_PCIE_LINK_START = 73,
-	PROGRESS_BOOT_ROM_PCIE_LINK_INIT = 74,
-	PROGRESS_BOOT_ROM_PCIE_PHY_SRAM_INIT_TIMEDOUT = 75,
-	PROGRESS_BOOT_ROM_PCIE_PHY_PARA_STS_CLR_TIMEDOUT = 76,
-	PROGRESS_BOOT_ROM_PCIE_LINK_LIMIT_GEN1 = 77,
-	PROGRESS_BOOT_ROM_PCIE_LINK_IATU_CONFIG = 78,
-	PROGRESS_BOOT_ROM_PCIE_LINK_FINISH = 79,
-	PROGRESS_BOOT_ROM_D2D_LINK_START = 80,
-	PROGRESS_BOOT_ROM_D2D_LINK_STS_BUSY = 81,
-	PROGRESS_BOOT_ROM_D2D_LINK_INTER_DIE_SYNC = 82,
-	PROGRESS_BOOT_ROM_D2D_LINK_INIT_TIMEDOUT = 83,
-	PROGRESS_BOOT_ROM_D2D_LINK_INIT_SUCCESS = 84,
-	PROGRESS_BOOT_ROM_RECOVERY_END = 85,
-	PROGRESS_BOOT_ROM_IN_RECOVERY_IMG = 86,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_EXCEEDS_MAX_SIZE = 87,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_DMA_START = 88,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_CRC_FAIL = 89,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_UNALIGNED_DEST = 90,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_INVALID_IMG_SIZE = 91,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_UNALIGNED_IMG_SIZE = 92,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_DMA_ERROR = 93,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_DMA_FINISH = 94,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_HDR_MAGIC_FAIL = 95,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_INVALID_IMG_OFFSET = 96,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_VERIFICATION_FAIL = 97,
-	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_FAIL = 98,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_START = 99,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_VERSION_FAIL = 100,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_KEY_REVOKED = 101,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_KEY_HASH_FAIL = 102,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_AUTH_FAIL = 103,
-	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_SUCCESS = 104,
-	PROGRESS_RECOVERY_IMG_IMG0 = 105,
-	PROGRESS_RECOVERY_IMG_IMG0_LOADADDR_INVALID = 106,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_STARTADDR_INVALID = 107,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_START = 108,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_CRC_FAILED = 109,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_UNALIGNED_DEST = 110,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_INVALID_IMG_SIZE = 111,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_UNALIGNED_IMG_SIZE = 112,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_ERROR = 113,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_FINISH = 114,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_VERIFICATION_FAIL = 115,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_START = 116,
-	PROGRESS_BOOT_REOCVERY_IMG_IMG0_FINISH = 117,
+	PROGRESS_BOOT_ROM_TPM_PCR_EXTEND_FAIL = 66,
+	PROGRESS_BOOT_ROM_TPM_FINISH = 67,
+	PROGRESS_PPBOOT_START = 68,
+	PROGRESS_PPBOOT_FINISH = 69,
+	PROGRESS_BOOT_ROM_IN_RECOVERY = 70,
+	PROGRESS_BOOT_ROM_IN_ROM_RECOVERY = 71,
+	PROGRESS_BOOT_ROM_PCIE_LINK_START = 72,
+	PROGRESS_BOOT_ROM_PCIE_LINK_INIT = 73,
+	PROGRESS_BOOT_ROM_PCIE_PHY_SRAM_INIT_TIMEDOUT = 74,
+	PROGRESS_BOOT_ROM_PCIE_PHY_PARA_STS_CLR_TIMEDOUT = 75,
+	PROGRESS_BOOT_ROM_PCIE_LINK_LIMIT_GEN1 = 76,
+	PROGRESS_BOOT_ROM_PCIE_LINK_IATU_CONFIG = 77,
+	PROGRESS_BOOT_ROM_PCIE_LINK_FINISH = 78,
+	PROGRESS_BOOT_ROM_D2D_LINK_START = 79,
+	PROGRESS_BOOT_ROM_D2D_LINK_STS_BUSY = 80,
+	PROGRESS_BOOT_ROM_D2D_LINK_INTER_DIE_SYNC = 81,
+	PROGRESS_BOOT_ROM_D2D_LINK_INIT_TIMEDOUT = 82,
+	PROGRESS_BOOT_ROM_D2D_LINK_INIT_SUCCESS = 83,
+	PROGRESS_BOOT_ROM_RECOVERY_END = 84,
+	PROGRESS_BOOT_ROM_IN_RECOVERY_IMG = 85,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_EXCEEDS_MAX_SIZE = 86,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_DMA_START = 87,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_CRC_FAIL = 88,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_UNALIGNED_DEST = 89,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_INVALID_IMG_SIZE = 90,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_UNALIGNED_IMG_SIZE = 91,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_DMA_ERROR = 92,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_DMA_FINISH = 93,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_HDR_MAGIC_FAIL = 94,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_INVALID_IMG_OFFSET = 95,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_VERIFICATION_FAIL = 96,
+	PROGRESS_INIT_RECOVERY_IMG_BOOT_DESC_FAIL = 97,
+	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_START = 98,
+	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_VERSION_FAIL = 99,
+	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_KEY_REVOKED = 100,
+	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_KEY_HASH_FAIL = 101,
+	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_AUTH_FAIL = 102,
+	PROGRESS_RECOVERY_IMG_BOOT_DESC_VERIFY_SUCCESS = 103,
+	PROGRESS_RECOVERY_IMG_IMG0 = 104,
+	PROGRESS_RECOVERY_IMG_IMG0_LOADADDR_INVALID = 105,
+	PROGRESS_BOOT_REOCVERY_IMG_IMG0_STARTADDR_INVALID = 106,
+	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_START = 107,
+	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_CRC_FAILED = 108,
+	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_UNALIGNED_DEST = 109,
+	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_INVALID_IMG_SIZE = 110,
+	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_UNALIGNED_IMG_SIZE = 111,
+	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_ERROR = 112,
+	PROGRESS_BOOT_REOCVERY_IMG_IMG0_DMA_FINISH = 113,
+	PROGRESS_BOOT_REOCVERY_IMG_IMG0_VERIFICATION_FAIL = 114,
+	PROGRESS_BOOT_REOCVERY_IMG_IMG0_START = 115,
+	PROGRESS_BOOT_REOCVERY_IMG_IMG0_FINISH = 116,
 };
 
 #endif /* _PROGRESS_BITS_H_ */
